src/jasper/helpers/forms.cr

```diff
@@ -64,39 +76,39 @@ module Jasper::Helpers::Forms
   # with collection Array(Array)
   def select_field(name : String | Symbol, collection : Array(Array), **options : Object)
     content(name: :select, options: options.to_h) do
-      collection.map{ |item| "<option value=\"#{item[0]}\">#{item[1]}</option>"}.join("")
+      collection.map { |item| "<option value=\"#{item[0]}\">#{item[1]}</option>" }.join("")
     end
   end
 
   def select_field(name : String | Symbol, collection : Array(Array))
     content(name: :select, options: {:name => name, :id => name}) do
-      collection.map{ |item| "<option value=\"#{item[0]}\">#{item[1]}</option>"}.join("")
+      collection.map { |item| "<option value=\"#{item[0]}\">#{item[1]}</option>" }.join("")
     end
   end
 
   # with collection Array(Hash)
   def select_field(name : String | Symbol, collection : Array(Hash), **options : Object)
     content(name: :select, options: options.to_h) do
-      collection.map{ |hash| "<option value=\"#{hash.first[0]}\">#{hash.first[1]}</option>"}.join("")
+      collection.map { |hash| "<option value=\"#{hash.first[0]}\">#{hash.first[1]}</option>" }.join("")
     end
   end
 
   def select_field(name : String | Symbol, collection : Array(Hash))
     content(name: :select, options: {:name => name, :id => name}) do
-      collection.map{ |hash| "<option value=\"#{hash.first[0]}\">#{hash.first[1]}</option>"}.join("")
+      collection.map { |hash| "<option value=\"#{hash.first[0]}\">#{hash.first[1]}</option>" }.join("")
```

Comment:
At the end of the day it's probably clean enough that way. However you probably should pipe the 2 values in instead of passing them in through and array called hash, then calling `hash[0]`.